### PR TITLE
Update logic to overwrite in data upload excel

### PR DESCRIPTION
**Pull request by danangmassandy**
This is for #815.

Preview message when there is overwriting existing data:
Screenshot_4766

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 7 min 26 sec

File: django_project/species/scripts/data_upload.py [hunk 1]
```diff
@@ -212,11 +213,13 @@ def finish(self, headers):
                               "".format(self.created_list)
 
         if self.existed_list > 0 and self.created_list == 0:
-            success_message = "{} rows already exist in the database." \
+            success_message = "{} rows already exist and have been " \
+                              "overwritten in the database." \
```
Comment (dimasciput) [inline django_project/species/scripts/data_upload.py:217]: Let's remove the database, as it can sometimes be too intimidating for non-technical users. :)

**Review by dimasciput — CHANGES_REQUESTED** [review]

File: django_project/species/scripts/data_upload.py (hunk 1 above)
Comment (danangmassandy) [reply django_project/species/scripts/data_upload.py:217]: sure, I'll update the text and fix the conflicts too

Comment (danangmassandy) [reply django_project/species/scripts/data_upload.py]: updated Dimas

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 7 min 23 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 22 sec

**Comment by LunaAsefaw** [comment]
"Success! 2  existing records have been overwritten." @dimasciput would that be more user friendly?

**Pull request merged**